tutorials/exporting-stackdriver-elasticcloud/index.md

```diff
@@ -0,0 +1,337 @@
+
+    ![Create an Elastic Cloud deployment](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/create_es_deployment.png)
+
+1.  While the deployment is finishing up, make sure to capture the credentials and store them in a safe place. While
+    the Cloud ID can be viewed from the deployment page, this is the only time the password for the elastic user is 
+    available. Visit the **Security** page to reset the password if needed. When considering production environments, create
+    new Elasticsearch credentials with tighter permissions and avoid using the `elastic` user. As
+    [documented](https://www.elastic.co/guide/en/cloud/master/ec-cloud-id.html): "On a production system, you should adapt 
+    these examples by creating a user that can write to and access only the minimally required indices."
+
+    ![Launching an Elastic Cloud deployment](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/es_deployment_launch.png)
+
+1.  Obtain the URI of the Elasticsearch endpoint that has been provisioned. A link to this endpoint can be copied from the 
+    **Deployments** page. This value will be needed to configure Logstash output plugin configuration.
+
+    ![Copy the Elasticsearch URI](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/copy_es_uri.png)
+
+    The next section provides steps to complete the setup to send events to the new Elasticsearch deployment.
+
+## Configure the Logstash VM
+
+1.  Navigate in the GCP console to the **Compute Engine > VM Instances** page. The GCP Console provides convenient access to
+    the VM with an embedded SSH console. The example shell commands in this tutorial are concatenated, but if issues are 
+    encountered when pasting multiple commands into this console, consider switching to
+    [OS login](https://cloud.google.com/compute/docs/instances/managing-instance-access#enable_oslogin).
+
+    ![SSH to VM](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/ssh_button.png)
+
+1.  Perform typical system updates and install OpenJDK:
+
+        sudo apt-get update && \
+        sudo apt-get -y upgrade && \
+        sudo apt -y install openjdk-8-jre-headless && \
+        echo "export JAVA_HOME=\"/usr/lib/jvm/java-8-openjdk-amd64\"" >> ~/.profile && \
+        sudo reboot
+
+    After a few moments, the VM will complete its reboot and can be accessed again via SSH.
+
+## Install Logstash
+
+1.  Install logstash from Elastic.
+
+        wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add - && \
+        echo "deb https://artifacts.elastic.co/packages/6.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-6.x.list && \
+        sudo apt-get update && sudo apt-get install logstash
+
+1.  Install the Logstash Plugin for Cloud Pub/Sub. Version 1.2.0 of the Logstash Pub/Sub plugin is
```

Comment:
I just checked, and it looks like version 1.2.1 is now available. I presume that that means that you need to update this section. I flagged a few instnace of references to specific versions above, too, just to make sure that you considered those.